src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/UpdateBatchCertificateRequestValidator.cs

```diff
@@ -16,6 +18,7 @@ public UpdateBatchCertificateRequestValidator(IStringLocalizer<BatchCertificateR
 
             RuleFor(m => m.CertificateReference).NotEmpty().WithMessage("Provide the certificate reference").DependentRules(() =>
             {
+                // TODO: Consider in the future how to merge both create & update versions as the Cert will always exist due to EPA 
```

Comment:
I think where these type of comments exist they should reference a Tech Debt ticket with the number.

Reply:
Added ticket numbers